<ui version="4.0" >
 <class>ListRegistroIvaBase</class>
 <widget class="QWidget" name="ListRegistroIvaBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Listados de IVA</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >
    <normaloff>:/Images/vat-list.svg</normaloff>:/Images/vat-list.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>48</horstretch>
         <verstretch>48</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/Images/vat-list.svg</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_crear" >
          <property name="minimumSize" >
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/document-new.svg</normaloff>:/Images/document-new.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_borrar" >
          <property name="minimumSize" >
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/edit-delete.svg</normaloff>:/Images/edit-delete.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>28</width>
            <height>28</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/view-refresh.svg</normaloff>:/Images/view-refresh.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>435</width>
            <height>29</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="textLabel1_2_2" >
       <property name="text" >
        <string>Fecha inicial:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BusquedaFecha" native="1" name="finicial" />
     </item>
     <item>
      <widget class="QLabel" name="textLabel1_4" >
       <property name="text" >
        <string>Fecha final:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BusquedaFecha" native="1" name="ffinal" />
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>71</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="TabWidget2" >
     <widget class="QWidget" name="tab1" >
      <attribute name="title" >
       <string>&amp;Sopotado</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
       <item>
        <widget class="SubForm2Bc" name="mui_tablasoportado" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2" >
      <attribute name="title" >
       <string>&amp;Repercutido</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
       <item>
        <widget class="SubForm2Bc" name="mui_tablarepercutido" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab3" >
      <attribute name="title" >
       <string>Res&amp;umen</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>5</number>
       </property>
       <property name="margin" >
        <number>5</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>IVA soportado:</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>5</number>
          </property>
          <property name="margin" >
           <number>5</number>
          </property>
          <item>
           <widget class="SubForm2Bc" name="mui_totalSoportado" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_3_3" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>IVA:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ivar" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_3_2_2" >
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Total:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_baseimpr" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>IVA repercutido:</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="spacing" >
           <number>5</number>
          </property>
          <property name="margin" >
           <number>5</number>
          </property>
          <item>
           <widget class="SubForm2Bc" name="mui_totalRepercutido" />
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>6</number>
            </property>
            <property name="margin" >
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="textLabel1_3" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>IVA:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_ivas" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="textLabel1_3_2" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Ignored" hsizetype="Ignored" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize" >
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text" >
               <string>Total:</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="m_baseimps" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment" >
               <set>Qt::AlignRight</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>BusquedaFecha</class>
   <extends>QWidget</extends>
   <header>../../../bulmalib/src/busquedafecha.h</header>
  </customwidget>
  <customwidget>
   <class>SubForm2Bc</class>
   <extends>QTableWidget</extends>
   <header>subform2bc.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections/>
</ui>
